scripts/add_electricity.py

```diff
@@ -272,16 +272,32 @@ def costs_for_storage(store, link1, link2=None, max_hours=1.0):
             dict(capital_cost=capital_cost, marginal_cost=0.0, co2_emissions=0.0)
         )
 
-    costs.loc["battery"] = costs_for_storage(
+    for max_hour in max_hours["battery"]:
+        costs.loc[f"battery {max_hour}h"] = costs_for_storage(
+            costs.loc["battery storage"],
+            costs.loc["battery inverter"],
+            max_hours=max_hour,
+        )
+    # cost for default 6h battery
+    costs.loc[f"battery"] = costs_for_storage(
         costs.loc["battery storage"],
         costs.loc["battery inverter"],
-        max_hours=max_hours["battery"],
+        max_hours=6,
```

Comment:
What do you think about avoiding hard coded values here ? We could add in the documentation that the first value will be the default and change code as follow:

```suggestion
        max_hours=max_hours["battery"][0],
```

Same applies for H2.